OsmAnd/src/net/osmand/plus/render/MapVectorLayer.java

```diff
@@ -105,54 +105,6 @@ public void onPrepareBufferImage(Canvas canvas, RotatedTileBox tilesRect, DrawSe
 		final MapRendererView mapRenderer = view.getMapRenderer();
 		if (mapRenderer != null && !oldRender) {
 			NativeCoreContext.getMapRendererContext().setNightMode(drawSettings.isNightMode());
-			OsmandSettings st = view.getApplication().getSettings();
```

Comment:
no code left in vector layer, so it means (most likely) if we disable all online layers vector layer will stop working.

``` mapRenderer.setVisualZoomShift(zoomMagnifier - 1.0f);``` disappeared from Pull Request???